population/population_type_metadata.go

```diff
@@ -0,0 +1,73 @@
+package population
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"github.com/ONSdigital/dp-api-clients-go/v2/clientlog"
+	dperrors "github.com/ONSdigital/dp-api-clients-go/v2/errors"
+	"github.com/ONSdigital/log.go/v2/log"
+	"github.com/pkg/errors"
+)
+
+type GetPopulationTypeMetadataInput struct {
```

Comment:
All exported structs and functions should have GoDoc comments explaining their purpose. See [other packages in this repo](https://github.com/ONSdigital/dp-api-clients-go/blob/main/image/image.go) for examples.